src/main/java/com/redhat/cloud/policies/app/rest/utils/PagingUtils.java

```diff
@@ -118,10 +118,22 @@ public static Pager extractPager(@NotNull UriInfo uriInfo) {
             if (filterMatcher.find()) {
                 String column = filterMatcher.group(1);
                 String value = queryParams.getFirst(key);
-                String operatorString = queryParams.getFirst(String.format("%s[%s]", FILTER_OP, column));
-                Filter.Operator operator = Filter.Operator.EQUAL;
-                if (operatorString != null) {
-                    operator = Filter.Operator.fromName(operatorString);
+                Filter.Operator operator;
+                if (column.equals("is_enabled")) {
+                    operator = Filter.Operator.BOOLEAN_IS;
+                    if (value==null || value.isEmpty()) {
+                        value = "true";
```

Comment:
+1

We could document that this value defaults to true when not found or empty:` @Parameter(name= filter[is_enabled]`

Reply:
I wonder if the @Param for filter:op[is_enabled] should just go away

Reply:
Ah, for this case it makes sense to just remove it, is the only possible value.